ui/src/app/views/admin.html

```diff
@@ -35,7 +35,7 @@
     </md-content>
   </div>
 
-  <div ng-if="!data.length == true">
+  <div ng-if="data.length == 0">
```

Comment:
@demon-36 I think div should be in else part of the **sign up request** because when all approve must show this part.
thank you : )